src/main/java/com/adorsys/keycloakstatuslist/service/StatusListService.java

```diff
@@ -15,6 +15,7 @@
 import org.apache.hc.core5.http.io.entity.EntityUtils;
 import org.apache.hc.core5.http.io.entity.StringEntity;
 import org.jboss.logging.Logger;
+import org.keycloak.jose.jwk.JWK; // Import added
```

Comment:
Is this comment relevant?
```suggestion
import org.keycloak.jose.jwk.JWK;
```